drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/columns/CallbackFactory.java

```diff
@@ -65,17 +67,23 @@
      * @param cell The Cell to be rendered.
      * @return
      */
-    public static <T, W extends ListBox, E extends MultiValueDOMElement<T, W>, F extends MultiValueSingletonDOMElementFactory<T, W, E>> Callback<E> makeOnCreationCallback( final F factory,
-                                                                                                                                                                            final GridCell<T> cell,
-                                                                                                                                                                            final Map<String, String> enumLookups ) {
-        return ( e ) -> {
+    public static <T, W extends ListBox, E extends MultiValueDOMElement<T, W>, F extends MultiValueSingletonDOMElementFactory<T, W, E>> Callback<E> makeOnCreationCallback(final F factory,
+                                                                                                                                                                           final GridCell<T> cell,
+                                                                                                                                                                           final Map<String, String> enumLookups) {
+        return (e) -> {
             final W widget = e.getWidget();
-            for ( Map.Entry<String, String> lookup : enumLookups.entrySet() ) {
-                widget.addItem( lookup.getValue(),
-                                lookup.getKey() );
+
+            if (widget.isMultipleSelect()) {
```

Comment:
Here's a change... if the ```ListBox``` is "multiple select" then show the lessor of the number of values or ```10```

Reply:
Looks reasonable